src/modules/workspaces/repositories/workspaces.repository.spec.ts

```diff
@@ -229,4 +230,75 @@ describe('SequelizeWorkspaceRepository', () => {
       });
     });
   });
+
+  describe('findWorkspaceUsers', () => {
+    it('When passing workspace id and not a search value, it should return all the users that belongs to that workspace', async () => {
+      const workspace = newWorkspace();
+      const user1 = newUser();
+      const user2 = newUser();
+      const workspaceUser1 = newWorkspaceUser({
+        workspaceId: workspace.id,
+        member: user1,
+        memberId: user1.uuid,
+      });
+      const workspaceUser2 = newWorkspaceUser({
+        workspaceId: workspace.id,
+        member: user2,
+        memberId: user2.uuid,
+      });
+      const mockWorkspaceUserModel = [
+        workspaceUser1,
+        workspaceUser2,
+      ] as unknown as WorkspaceUserModel[];
+
+      const spyWUM = jest
+        .spyOn(workspaceUserModel, 'findAll')
+        .mockResolvedValueOnce(mockWorkspaceUserModel);
+
+      const result = await repository.findWorkspaceUsers(workspace.id);
+
+      expect(result).toBeInstanceOf(Array as unknown as WorkspaceUser[]);
```

Comment:
Instead of this and a match by email, use strictEqual to match the entire array with the result